File: back/src/main/java/com/integradora/back/controller/detalleorden/dto/DetalleOrdenDTO.java
package com.integradora.back.controller.detalleorden.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class DetalleOrdenDTO {
    private Long id;           // id del DetalleOrden (para cambios de estado desde cocina)
    private Long platilloId;
    private String nombre;
    private Integer cantidad;
    private BigDecimal precioUnitario;
    private String nota;
    private String estadoPreparacion;
}

File: mobile/app/src/main/java/proyecto/personal/proyectointegradorii/data/remote/dto/detalleorden/DetalleOrdenDTO.kt
package proyecto.personal.proyectointegradorii.data.remote.dto.detalleorden

data class DetalleOrdenDTO(
    val id: Long,
    val platilloId: Long?,
    val nombre: String?,
    val cantidad: Int,
    val precioUnitario: Double,
    val nota: String?,
    val estadoPreparacion: String?
)
